.travis.yml

```diff
@@ -1,17 +1,35 @@
-language: python
-sudo: required
-dist: trusty
+matrix:
+  include:
+    - language: python
+      sudo: required
+      dist: trusty
 
-python:
-  - "3.5"
+      python: 3.5
 
-install:
-  - pip3 install codecov pytest pytest-cov
-  - ./utils/jumspcale_install.sh
-  - ./utils/zrobot_install.sh
+      install:
+        - pip3 install codecov pytest pytest-cov
+        - ./utils/jumspcale_install.sh
+        - ./utils/zrobot_install.sh
 
-script:
-  - make test
+      script:
+        - make test
 
-after_success:
-  - codecov
\ No newline at end of file
+      after_success:
+        - codecov
+
+    - language: python
+      sudo: true
+      dist: trusty
+      if: type IN (api, cron)
+
+      python: 3.5
+
+      install:
+        - pip3 install git+https://github.com/gigforks/packet-python.git
```

Comment:
why do you use this ? I don't see this is needed anymore, our fix on this lib have been merge upstream month ago

Reply:
ok .. will check this